lib/broadside/deploy/ecs.rb

```diff
@@ -326,6 +338,10 @@ def create_new_task_revision
       task_def
     end
 
+    def service_exists?
+      !ecs_client.describe_services({ cluster: config.ecs.cluster, services: [family] }).failures.empty?
```

Comment:
the AWS services don't really return enumerables, they return weird `Seahorse` data structures with a `.failures` section.